Packs/HPE_Service_Manager/Integrations/integration-HPE_Service_Manager.yml

```diff
@@ -83,83 +75,210 @@ script:
                 Title: args.title,
             }
         };
-
-        if (args.customFields !== undefined){
-            fields = JSON.parse(args.customFields);
-            for (var field in fields) {
-                newIncident['Incident'][field] = fields[field]
-            }
```

Comment:
keep this block to handle customFields fields and remove the other fields you added